scripts/consensus_merge.py

```diff
@@ -300,13 +273,14 @@ def write_output_header(output_vcf, sample_list, contig_list, hotspot_source=Non
     output_vcf.header.formats.add('DPR', '1', 'Integer',
             'Difference between highest and lowest DP')
     for contig in sorted(contig_list, key=lambda x: x.id):
-        if strip_chr(contig.name) in ALLOWED_CHROMS:
+        if contig.name in ALLOWED_CHROMS:
```

Comment:
That would be cool. As it is, I could use the Strelka2 header to make a list of 'known contigs' and then raise an error if any provided contig is not on this list, which would at least alert the user to possible errors. As it is, if you miscapitalize a name in the input, it will just silently drop all the calls on the chromosome you thought you were specifying.